Sources/minify/src/JS.php

```diff
@@ -15,7 +15,7 @@
 class JS extends Minify
 {
     /**
-     * Var-matching regex based on http://stackoverflow.com/a/9337047/802993.
+     * Var-matching regex based on https://stackoverflow.com/a/9337047/802993.
```

Comment:
Just to add some meta to this since it came across my feeds.  https://nickcraver.com/blog/2017/05/22/https-on-stack-overflow/
They have enabled it completely.